applications/CompressiblePotentialFlowApplication/python_scripts/compute_forces_on_nodes_process.py

```diff
@@ -0,0 +1,58 @@
+import KratosMultiphysics
+import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
+
+def Factory(settings, Model):
+    if(not isinstance(settings, KratosMultiphysics.Parameters)):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string")
+    return ComputeForcesOnNodesProcess(Model, settings["Parameters"])
+
+# all the processes python processes should be derived from "python_process"
+
+class ComputeForcesOnNodesProcess(KratosMultiphysics.Process):
+    def __init__(self, Model, settings):
+        KratosMultiphysics.Process.__init__(self)
+
+        default_parameters = KratosMultiphysics.Parameters(r'''{
+            "model_part_name": "please specify the model part that contains the surface nodes",
```

Comment:
```suggestion
            "model_part_name": "",
```
By setting the default model part name to an empty string you can easily check below if it has been provided or not (this is just a suggestion).

Reply:
Thank You. I did the modification and now it is all set to be merged with master